build/config.props

```diff
@@ -37,6 +37,35 @@
     <VsTargetChannel Condition="'$(IsEscrowMode)' == 'true'">int.d$(VsTargetMajorVersion).$(MinorNuGetVersion)</VsTargetChannel>
   </PropertyGroup>
 
+ <PropertyGroup Condition=" '$(PreReleaseVersion)' == '' ">
```

Comment:
This is just step 1. 

You basically can get the exact version from buildinfo.json, which is generated here: https://github.com/NuGet/NuGet.Client/blob/0ebb20188285663d7c8349df9ddfd71020c231c4/scripts/cibuild/ConfigureVstsBuild.ps1#L170-L189.

Reply:
You need an equivalent target as the SDKVSSemanticVersion and the likes

Reply:
If I can get the version there, why do I need this change?

Reply:
The targets and values are defined in config.props for all other properties. common.project.props has other imports and properties set that you don't want or need.

Reply:
How's this?
![image](https://github.com/NuGet/NuGet.Client/assets/49205731/e24b7582-b86e-4cd8-a0f5-1e10b87a0631)

Reply:
I think that's what you want.